### PR TITLE
Add convenience initializer for backwards compatibility #trivial

### PR DESCRIPTION
**Pull request by wiseoldduck**

### CODE REVIEW

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Any source code changes should have an entry in CHANGELOG.md or have #trivial in their title.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Any source code changes should have an entry in CHANGELOG.md or have #trivial in their title.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Source/Classes/PINURLSessionManager.m [hunk 1]
```diff
@@ -48,6 +48,13 @@ - (void)invalidateSessionAndCancelTasks
     [self unlock];
 }
 
+- (nonnull NSURLSessionDataTask *)dataTaskWithRequest:(nonnull NSURLRequest *)request
+                                    completionHandler:(nonnull PINURLSessionDataTaskCompletion)completionHandler 
+{
+    return [self dataTaskWithRequest:request priority:PINRemoteImageManagerPriorityDefault
```
Comment (nguyenhuy) [inline Source/Classes/PINURLSessionManager.m]: Let's put `priority:PINRemoteImageManagerPriorityDefault` in a new line.

Comment (wiseoldduck) [reply Source/Classes/PINURLSessionManager.m]: It is done :)

**Pull request merged**